lambda-src/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Sends a message to a Slack channel",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "_mocha -u bdd --timeout 999999 --colors ../test"
```

Comment:
Is it not `mocha` rather than `_mocha`?

Reply:
...yes
you might like wrap in istanbul too for code coverage, not a now thing or a requirement unless you really feel like it

Reply:
hmm... both `_moca` and `moca` work. Not sure what the difference is between them. Got rid of the underscore anyway